### PR TITLE
Mariocube.MFAudio version 11

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110298&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/m/Mariocube/MFAudio/11/Mariocube.MFAudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.MFAudio
+PackageVersion: 11
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Other%20Consoles/PlayStation/MFAudiov11.zip
```
Comment (Trenly) [inline manifests/m/Mariocube/MFAudio/11/Mariocube.MFAudio.installer.yaml:7]: This is not an official mirror. The original publisher appears to be linked to on the Zophar site.

https://www.zophar.net/utilities/ps2util/mfaudio-1-1.html

File: manifests/m/Mariocube/MFAudio/11/Mariocube.MFAudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.MFAudio
+PackageVersion: 11
+PackageLocale: en-US
+Publisher: Mariocube
+PackageName: MFAudio
+License: This tool allows you to play, convert, and downsample audio files that use some of the more common formats among PS2 games.
```
Comment (Trenly) [inline manifests/m/Mariocube/MFAudio/11/Mariocube.MFAudio.locale.en-US.yaml:9]:
```suggestion
License: Freeware
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by upintheairsheep** [comment]
All former Mariocube packages are to be changed to their real publisher

**Comment by upintheairsheep** [comment]
Improper source, will recreate

**Pull request closed without merging**